test/test_api_usage.py

```diff
@@ -2412,18 +2412,18 @@ def test_trychangevisibility(self):
         # Change the subscription of the namespace.
         self.putJsonResponse(UserPlan, data=dict(plan="personal-2018"))
 
-        # Try to make private.
+        # Try to make private. Superusers with full access bypass license limits.
         self.postJsonResponse(
             RepositoryVisibility,
             params=dict(repository=self.SIMPLE_REPO),
             data=dict(visibility="private"),
-            expected_code=402,
+            expected_code=200,
```

Comment:
isn't this technically removing the previous test where we check for plan limits?

Reply:
Right - because a `root` user shouldn't be restricted by license limitations. They have full access to do whatever. See parent comment above for details.